custom_components/hacs/operational/factory.py

```diff
@@ -34,7 +34,10 @@ async def safe_update(self, repository):
             try:
                 await repository.update_repository()
             except (AIOGitHubAPIException, HacsException) as exception:
-                _LOGGER.error("%s - %s", repository.data.full_name, exception)
+                if str(exception) == "Repository is archived.":
```

Comment:
This kind of hack is too prone to fail, it needs to target a custom exception, like `HacsRepositoryArchived` or something.

Reply:
To future-proof it, would it be better to call it `HacsWarningException`?

Reply:
That does not make sense, the exceptions should not dictate how it's logged.

Reply:
Okay. I've updated it.